apps/frontend/src/app/5_pages/LiquidityBookPage/components/AddLiquidityModal/Content.tsx

```diff
@@ -0,0 +1,217 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+
+import { t } from 'i18next';
+
+import { LiquidityDistribution } from '@sovryn/joe-sdk-v2';
+import {
+  AmountInput,
+  Button,
+  ButtonStyle,
+  ErrorBadge,
+  ErrorLevel,
+  FormGroup,
+} from '@sovryn/ui';
+
+import { AssetRenderer } from '../../../../2_molecules/AssetRenderer/AssetRenderer';
+import { CurrentStatistics } from '../../../../2_molecules/CurrentStatistics/CurrentStatistics';
+import { MaxButton } from '../../../../2_molecules/MaxButton/MaxButton';
+import { useAmountInput } from '../../../../../hooks/useAmountInput';
+import { useAssetBalance } from '../../../../../hooks/useAssetBalance';
+import { useCurrentChain } from '../../../../../hooks/useChainStore';
+import { translations } from '../../../../../locales/i18n';
+import { decimalic } from '../../../../../utils/math';
+import { LiquidityBookPool } from '../../LiquidityBookPage.types';
+import { useHandleLiquidity } from '../../hooks/useHandleLiquidity';
+import { LiquidityPriceForm } from './LiquidityPriceForm';
+
+type ContentProps = {
+  pool: LiquidityBookPool;
+  onClose: () => void;
+};
+
+export const Content: FC<ContentProps> = ({ pool, onClose }) => {
+  const chainId = useCurrentChain();
+  const baseSymbol = pool.pair[0].symbol!;
+  const quoteSymbol = pool.pair[1].symbol!;
+
+  const { balance: baseBalance } = useAssetBalance(baseSymbol, chainId);
+  const { balance: quoteBalance } = useAssetBalance(quoteSymbol, chainId);
+
+  const [baseAmount, onBaseAmountChange] = useAmountInput('');
+  const [quoteAmount, onQuoteAmountChange] = useAmountInput('');
+
+  const [shape, setShape] = useState<LiquidityDistribution>(
+    LiquidityDistribution.SPOT,
+  );
+
+  const [min, setMin] = useState<number>(pool.activeBinId - 25);
+  const [max, setMax] = useState<number>(pool.activeBinId + 25);
```

Comment:
```suggestion
  const [min, setMin] = useState(pool.activeBinId - 25);
  const [max, setMax] = useState(pool.activeBinId + 25);
```
You don't need to specify the types here.